enterprise/server/secrets/secrets.go

```diff
@@ -113,6 +113,10 @@ func (s *SecretService) ListSecrets(ctx context.Context, req *skpb.ListSecretsRe
 	return rsp, nil
 }
 
+// UpdateSecret updates the secret with the given name associated with the group
+// the user is authenticated as, creating it if it does not exist. It returns,
+// in order, the response proto, whether the secret was newly created or not,
+// and any error encountered.
```

Comment:
unrelated?

Reply:
mmm, good call. I must have added this comment while reading code and then forgotten to make its own doc PR. i'll take it out.